tests/copyright_check/config.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+import sys
+import os
+
+
+debug_logs = False
+print_logs = sys.stderr if debug_logs else open("/dev/null", "w")
+
+validator_path = "tests/copyright_check"
+default_sample_file = "copyright_signature.py"
+
+rootdir = os.path.abspath(os.path.dirname(__file__) + "/../../")
+default_check_dir = os.path.join(rootdir, validator_path)
```

Comment:
The presence of `/` into the path assumes unix-like platform ... is this really needed?

Have you tried to consider to have something like the suggestion.
It would allow to prevent platform dependency
```suggestion
from pathlib import Path

default_sample_file = "copyright_signature.py" 
rootdir = Path(__file__).parent.parent.absolute()
default_check_dir = rootdir / "tests" / "copyright_check"
```

NOTE: The type of `rootdir` and `default_check_dir` will now be `pathlib.Path` instead of `str`